src/core/lombok/core/Versions.java

```diff
@@ -78,7 +78,7 @@ public static String getFullVersion() {
 		String version = String.format("v%s \"%s\"", VERSION, RELEASE_NAME);
 		if (!isEdgeRelease()) return version;
 		
-		InputStream in = Version.class.getResourceAsStream("/release-timestamp.txt");
+		InputStream in = Versions.class.getResourceAsStream("/release-timestamp.txt");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code contains a resource that might not be closed properly. This can cause a resource leak that slows down or crashes your system.

**Fix**
Consider closing the following resource: *in*. The resource is referenced at lines 82, 87. The resource is closed at line 96. There are other execution paths that do not contain closure statements. Ensure *in* is closed in a try-finally block or declared in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).